app/routers/auth.py

```
@@ -51,6 +55,90 @@ async def kakao_callback(code: str = Query(...), db: AsyncSession = Depends(get_
         raise HTTPException(status_code=400, detail=f"Login failed: {str(e)}")
 
 
+### ✅ Google OAuth 개선 (보안 강화)
+@router.get("/oauth/google/login")
+async def google_login_redirect(request: Request):
+    """구글 로그인 페이지로 리디렉션"""
+    import secrets
```

Ruff (0.8.2)

Redefinition of unused `secrets` from line 1

Remove definition: `secrets`

(F811)